elixir/README.md

```diff
@@ -7,10 +7,11 @@ The applications in this directory make use of the [AtomVM Mix Plugin](https://g
 
 Some example programs can only run on specific platform.  The following table summarizes the platforms on which the Elixir example programs can be run.
 
-| Example Program | esp32 | stm32 | generic_unix |
-|-----------------|-------|-------|--------------|
-| Blinky          | ✅ | ✅ | ❌ |
-| HelloWorld      | ✅ | ✅ | ✅ |
+| Example Program | esp32 | stm32 | Pico | Pico W | generic_unix |
+|-----------------|-------|-------|------|--------|--------------|
+| Blinky          | ✅ | ✅ | ✅ | ✅ | ❌ |
```

Comment:
For Blinky to work out of the box on STM32 we need to add a platform check to determine the pin to use, since STM32 pins need to be a tuple. I have been meaning to go back and make that change. I can do that in a separate PR.